Documentation/Metrics/arangodb_ioheartbeat_failures_remove_total.yaml

```diff
@@ -0,0 +1,19 @@
+name: arangodb_ioheartbeat_failures_remove_total
+introducedIn: "3.8.7"
+help: |
+  Number of failures in the io heartbeat remove test.
```

Comment:
General question: do we need different metrics for writes, removes and reads? Wouldn't it be sufficient to see if _one_ of the operations fails?

Reply:
This is probably a matter of debate. I wanted to have visibility if only certain aspects of the underlying volume's functionality stopped working. For example, if writing would still be working but reading no longer or incorrectly. This is maybe a bit paranoid and rather unlikely. I think I will reduce it to one failure counter and one delay counter as reaction to this comment, but will then leave the different log messages at INFO level in, so that they would be visible in the logs at the default logging level. How does that sound?

Reply:
Done.